### ISSUE TITLE
Overflow menu focus behavior

### ISSUE THREAD

**Issue by juuso-vaadin** [vaadin-component-factory/vcf-breadcrumb#12]
There is an accessibility issue with Vaadin Popover being a focus target. Focus ring is not visible in Lumo theme.
The ideal solution would be to not have the Popover element focusable, but rather the first element focused by default when Popover is opened.

<img width="487" height="320" alt="Image" src="https://github.com/user-attachments/assets/3f223ead-6470-44c6-a4db-945ab6951d4c" />
